CHANGELOG.md

```diff
@@ -31,6 +31,9 @@
 
 - Change: Failure to report metrics is logged using loglevel info rather than error.
 
+- Bugfix: A potential deadlock situation is fixed that sometimes caused the user daemon to become a zombie process when the user
+  was logged in. 
```

Comment:
I normally understand the term "zombie process" in the unix sense: "a process that has exited but whose parent has not yet `wait()ed` on it".